packages/stylelint-plugin/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@sourcegraph/stylelint-plugin-sourcegraph",
+  "version": "1.0.0",
+  "description": "Custom stylelint rules for the main Sourcegraph repo",
```

```suggestion
  "description": "Custom stylelint rules for Sourcegraph",
```

Is there any reason we shouldn't keep rules here generic and anything specific config/overrides should live in the main Sourcegraph repo?

Kinda like how we have the generic [eslint-config repo](https://github.com/sourcegraph/eslint-config) and specific overrides in `eslintrc` files in the main repo

---

Yeah, that's the intention — rules in this repo will be generic. Here, I was trying to convey that the primary consumer of these rules will be the main Sourcegraph repo. Happy to update the description 👍